libs/api/src/lib/media/novaMediaAdapter.ts

```diff
@@ -121,6 +129,7 @@ export class NovaMediaAdapter implements MediaAdapter {
       )
     }
 
+    /** NOT USED AT THE MOMENT WE SHOULD NOT HAVE UNUSED FUNCTIONS AS DDOS SURFACE THEY NEED TO BE LIMITED BY CONFIGURATION SAME AS QUALITY AND DIMENSION!
```

Comment:
is this wanted?

Reply:
or can we delete unused code?

Reply:
No i want to keep it for karl. If he needs at some point he does not have to add them again